backend/hct_mis_api/apps/web/templates/web/index.html

```diff
@@ -21,6 +21,25 @@
     {% endif %}
   </script>
 
+  {% if MATOMO_TRACKER_URL and MATOMO_SITE_ID and MATOMO_SCRIPT_URL %}
```

Comment:
can't this and the thing added in `backend/hct_mis_api/apps/administration/templates/admin/base.html` be a template and imported like this?

```
{% include "matomo.html" %}
```

Reply:
Good idea, I will change it 👍